CTFd/admin.py

```diff
@@ -452,10 +455,11 @@ def admin_team(teamid):
     elif request.method == 'POST':
         admin_user = request.form.get('admin', None)
         if admin_user:
-            admin_user = 1 if admin_user == "true" else 0
+            admin_user = True if admin_user == 'true' else False
             user.admin = admin_user
-            user.banned = admin_user
+            #user.banned = admin_user
```

Comment:
Ok, it just looked funny.  Perhaps there should be a comment, as it wasn't clear?

I'll add.